### PR TITLE
Qrng review march15

### PR DESCRIPTION
**Pull request by KenCarvAPI**
Refresh to explore section

### CODE REVIEW

File: docs/explore/qrng/index.md [hunk 1]
```diff
@@ -23,33 +24,40 @@ Smart contract platforms typically emulate a deterministic virtual machine,
 which cannot generate random numbers. In such cases, random number generation
 (RNG) needs to be provided as an oracle service.
 
-Most computer hardware is also designed to run deterministically, which means
-regular computers cannot generate true random numbers. Instead, pseudorandom
-number generation (PRNG) methods are used to generate _adequately
-random-looking_ numbers. Unless stated otherwise, RNG oracle services should be
-expected to be PRNG-based.
+There are are currently limited options to access entropy on-chain. API3 Quantum
```
Comment (vanshwassan) [inline docs/explore/qrng/index.md:27]:
```suggestion
There are currently limited options to access entropy on-chain. API3 Quantum
```

**Review by vanshwassan — CHANGES_REQUESTED** [review]

File: docs/explore/qrng/QRNG-providers.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+> numbers are generated can be found in
+> [Appl. Phys. Lett. 98, 231103 (2011)](https://dx.doi.org/10.1063/1.3597793)
+> and
+> [Phys. Rev. Applied 3, 054004 (2015)](https://dx.doi.org/10.1103/PhysRevApplied.3.054004).
+
+### Quintessence
+
+In a world where data is too important to be compromised, QuintessenceLabs
+delivers powerful data protection, enhanced by the quantum power of
+nature.QuintessenceLabs uses quantum tunneling to deliver truly random numbers
```
Comment (dcroote) [inline docs/explore/qrng/QRNG-providers.md:62]: Missing space after period

File: docs/explore/qrng/index.md [hunk 3]
```diff
@@ -59,51 +67,10 @@ awkward case where it is not possible to talk of an ownership of the data. Here,
...
-> appearing and disappearing all the time. This is because the vacuum still
-> possesses a zero-point energy. Consequently, the electromagnetic field of the
-> vacuum exhibits random fluctuations in phase and amplitude at all frequencies.
-> By carefully measuring these fluctuations, we are able to generate ultra-high
-> bandwidth random numbers. <br/><br/> The technical details on how the random
-> numbers are generated can be found in
-> [Appl. Phys. Lett. 98, 231103 (2011)](https://dx.doi.org/10.1063/1.3597793)
-> and
-> [Phys. Rev. Applied 3, 054004 (2015)](https://dx.doi.org/10.1103/PhysRevApplied.3.054004).
+<!-- review -->
```
Comment (dcroote) [inline docs/explore/qrng/index.md:73]: Delete these comments?

File: docs/explore/qrng/index.md [hunk 4]
```diff
@@ -121,10 +88,4 @@ cheaper, and more customizable request fulfillments. In addition, integrations
 to more chains and QRNG providers will be made available over time, and
 higher-level QRNG services will be built leveraging these.
 
-See these guides and learn how to use QRNG:
```
Comment (dcroote) [inline docs/explore/qrng/index.md:124]: Why not link to the QRNG [getting started guide page](https://docs.api3.org/guides/qrng/)? It seems like we'd want to have an actionable next step after viewing this Explore page.

File: docs/reference/qrng/chains.md [hunk 5]
```diff
@@ -83,11 +93,7 @@ beyond one level deep inside and element (i.e. <div>). See /dev/tabs.md
 to not break as well.
 -->
 
-## ANU
-
-ANU is only available on mainnets.
-
-### Mainnets
+## ANU Mainnet
```
Comment (dcroote) [inline docs/reference/qrng/chains.md:96]: Shouldn't this (and the other similar headings) be plural "Mainnets"?

**Comment by KenCarvAPI** [comment]
@vanshwassan @dcroote thanks for reviewing. Implemented the amends/thoughts. 

@Ashar2shahid this is ready for you to review and merge if happy.

File: docs/explore/qrng/index.md [hunk 6]
```diff
@@ -88,4 +83,9 @@ cheaper, and more customizable request fulfillments. In addition, integrations
 to more chains and QRNG providers will be made available over time, and
 higher-level QRNG services will be built leveraging these.
 
+# Get Started with QRNG
+
+Follow [this guide](https://docs.api3.org/guides/qrng/) to get started with QRNG
```
Comment (dcroote) [inline docs/explore/qrng/index.md:88]: Nit: change to internal link i.e. no https...

File: docs/reference/qrng/chains.md [hunk 7]
```diff
@@ -93,7 +93,7 @@ beyond one level deep inside and element (i.e. <div>). See /dev/tabs.md
 to not break as well.
 -->
 
-## ANU Mainnet
+## ANU Mainnets Chain ID
```
Comment (dcroote) [inline docs/reference/qrng/chains.md:96]: I think this should just be "ANU Mainnets" - the tables have more than just chain IDs (same applies to the other headings).

**Comment by wkande** [comment]
@dcroote Should we close this PR? QRNG being a thing of the past and all.

**Comment by dcroote** [comment]
I'm good with closing

**Comment by Ashar2shahid** [comment]
yes closing

**Pull request closed without merging**